cluster/openstack-heat/util.sh

```diff
@@ -32,7 +32,7 @@ fi
 # Verify prereqs on host machine
 function verify-prereqs() {
  # Check the OpenStack command-line clients
- for client in swift glance nova heat;
+ for client in swift glance nova openstack;
```

Comment:
Discussed with @elsonrodriguez on Slack but I believe you still need Heat here or it results in:

openstack: 'stack' is not an openstack command. See 'openstack --help'.
Did you mean one of these?
  backup create
  backup delete
  backup list
  backup restore
  backup show